tests/input_data/test_apbs_setup-1fas-ion.json

```diff
@@ -0,0 +1,52 @@
+{
+    "type": "mg-auto",
+    "calcenergy": "total",
+    "calcforce": "no",
+    "output_scalar": [
+    "writepot"
+    ],
+    "writeformat": "dx",
+    "hiddencheck": "local",
+    "mol": "1",
+    "pdb2pqrid": "sampleId",
+    "cgcent": "mol",
+    "fgcent": "mol",
+    "solvetype": "lpbe",
+    "bcfl": "sdh",
+    "chgm": "spl2",
+    "srfm": "smol",
+    "charge0": "1",
+    "charge1": "-1",
+    "charge2": "",
+    "conc0": "0.15",
+    "conc1": "0.15",
+    "conc2": "",
+    "radius0": "0.95",
```

Comment:
APBS can't distinguish between ions with different radii. Can we please use 2.0 for both ion radii?

Reply:
The test file's been updated to use `2.0` for this radius